app/src/main/res/values-night/colors.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="colorPrimaryDark">#f2734c</color>
+    <color name="colorPrimary">@color/orangish</color>
+    <color name="colorAccent">@color/veryLightPink</color>
+    <color name="black">#333333</color>
+    <color name="grey850">#343434</color>
+    <color name="salmon">#ff6666</color>
+    <color name="white_transparency">#b9ffffff</color>
+    <color name="white_transparent">#00ffffff</color>
+    <color name="white">#ffffff</color>
+    <color name="dark_white">#E8E8E8</color>
+    <color name="battleship_gray">#64707b</color>
+    <color name="green">#8dbf75</color>
+    <color name="algae_green">#25d366</color>
+
+    <color name="dark_grey">#222222</color>
+    <color name="neon_red">#ff003a</color>
+    <color name="light_transparent">#16000000</color>
+
+    <!--    New Colors-->
+    <color name="pastel_red">#e74e4b</color>
```

Comment:
and if colors are the same for both the mode then, in that case, they shouldn't be declared again and again